fuzz/fuzz_targets/splines.rs

```diff
@@ -0,0 +1,133 @@
+use honggfuzz::fuzz;
```

Comment:
Same question here - can we guarantee the same rand will be generated each time?